Update "Lutris (Wine)" with more recent info

ultra-azu opened the pull request:
The wiki mentions Origin which (afaict) is now deprecated by EA. Also, the EA App's overlay causes Northstar to hang, which isn't mentioned in the wiki.

If you want, I can also modify the article to mention this lutris script https://github.com/begin-theadventure/lutris-scripts/tree/main/lutris-scripts/Northstar/EA%20App, although I didn't need it for setting up Northstar.

itscynxx commented:
> If you want, I can also modify the article to mention this lutris script https://github.com/begin-theadventure/lutris-scripts/tree/main/lutris-scripts/Northstar/EA%20App, although I didn't need it for setting up Northstar.

I would personally do this, I've been meaning to but never have. I think having the scripts easily accessible on the wiki page is better than the majority of Lutris users that need this script and have to ask in the discord for someone to send the tag for it

itscynxx commented on docs/steamdeck-and-linux/installing-on-steamdeck-and-linux.md:
```diff
@@ -34,14 +34,17 @@ If you're using an older version of NorthstarProton, replace the information abo
-4. **If you have the game installed on Lutris:** right click _Titanfall 2_ > _Configure_ > _Game Options_ > Set _Executable path_ to _NorthstarLauncher.exe_
-5. **Otherwise:** click the `+` button in the top left > set the name to whatever and _Runner_ to _Wine_ > click on _Game options_ > set _Executable path_ to _NorthstarLauncher.exe_ then save.
+2. Install the latest version of Northstar using [FlightCore](../installing-northstar/northstar-installers#geckoeidechse-flightcore), [Viper](../installing-northstar/northstar-installers#0negal-viper), or do it manually
+   1. For manual install download the latest version of Northstar from the [releases](https://github.com/R2Northstar/Northstar/releases) page
```
```suggestion
   1. For manual install, download the latest version of Northstar from the [releases](https://github.com/R2Northstar/Northstar/releases) page
```

itscynxx replied:
Minor grammar nitpick

itscynxx commented on docs/steamdeck-and-linux/installing-on-steamdeck-and-linux.md:
```diff
@@ -34,14 +34,17 @@ If you're using an older version of NorthstarProton, replace the information abo
+   1. For manual install download the latest version of Northstar from the [releases](https://github.com/R2Northstar/Northstar/releases) page
+   2. Then extract all contents of the file to your Titanfall 2 folder ( Right click _Titanfall 2_ > Click _Browse files_ )
+3. Add `%command% -northstar` as a [launch option](../installing-northstar/troubleshooting.md#launch-opts) to Titanfall2
+4. Disable in-game overlay: In EA App, click on hamburger menu at top left > Click _Settings_ > Click _Application_ > Toggle _in-game overlay_
```
```suggestion
4. Disable in-game overlay: In EA App, click on the hamburger menu on the top left > Click _Settings_ > Click _Application_ > Toggle _in-game overlay_ to OFF
```

itscynxx replied:
Minor grammar nitpicks, clarify on toggling overlay to off

CooldudePUGS reviewed (commented):
I've got some minor grammatical nitpicks but apart from that and preferably adding the Lutris script to the wiki, looks good

GeckoEidechse reviewed (commented):
I would not fully remove mentions of Origin but simply mention both. The reason being that I still see a bunch of people using Origin with Lutris.

Clearly Origin is on the way out and there's no reason to keep clinging to it but I'd still keep it as a footnote at the very least ^^

ultra-azu commented:
Done for now. Should I keep the mention of the official Lutris script or do I remove it?

GeckoEidechse commented:
> Done for now. Should I keep the mention of the official Lutris script or do I remove it?

You mean this one? https://lutris.net/games/titanfall-2/
Honestly no idea how well it works as I've never used Lutris. Maybe @CooldudePUGS or @stygia-dev can chime in on that ;)

Also as a sidenote, please don't force push. We squash merge branches anyway, so any "fixup" or "revert" commits will be cleaned away on merge. Force pushing makes it more difficult for a reviewer like me to see what the changes were ^^"

GeckoEidechse commented on docs/steamdeck-and-linux/installing-on-steamdeck-and-linux.md:
```diff
@@ -33,15 +33,20 @@ If you're using an older version of NorthstarProton, replace the information abo
-3. Extract all contents of the file to your Titanfall 2 folder
-4. **If you have the game installed on Lutris:** right click _Titanfall 2_ > _Configure_ > _Game Options_ > Set _Executable path_ to _NorthstarLauncher.exe_
-5. **Otherwise:** click the `+` button in the top left > set the name to whatever and _Runner_ to _Wine_ > click on _Game options_ > set _Executable path_ to _NorthstarLauncher.exe_ then save.
+1. If you don't already have the game downloaded, You can use [this Lutris script](https://github.com/begin-theadventure/lutris-scripts/tree/main/lutris-scripts/Northstar/EA%20App), or [this one](https://lutris.net/games/titanfall-2/).
```
typo

```suggestion
1. If you don't already have the game downloaded, you can use [this Lutris script](https://github.com/begin-theadventure/lutris-scripts/tree/main/lutris-scripts/Northstar/EA%20App), or [this one](https://lutris.net/games/titanfall-2/).
```

itscynxx commented:
> You mean this one? https://lutris.net/games/titanfall-2/

No this is for vanilla and doesn't usually work for people for Northstar (from what I see from helping at least)
The issue is im not entirely sure how the NS script works. I didn't think it installed the game again and instead just installed Northstar but I honestly don't know. Would be 5 hours before I could test, and asking the creator makes it seem like it reinstalls but im not entirely sure

lizardwizardonthewall commented:
> Done for now. Should I keep the mention of the official Lutris script or do I remove it?

Remove it. If we're going to fully support Lutris going forward as an alternative to Proton, we should be suggesting begin-theadventure's script, so long as it actually installs Titanfall2 with it.

ultra-azu commented:
Added the typo fix and removed the official script. Tbh I haven't tested begin-theadventure's script, but it pretty much does the same thing as the official one except it adds a dll override for wsock32 and removes ActivationUI.exe from ExcludeProcesses

GeckoEidechse commented:
@CooldudePUGS @stygia-dev if any of you says this looks good after changes, I'm happy to merge ^^

lizardwizardonthewall commented on docs/steamdeck-and-linux/installing-on-steamdeck-and-linux.md:
```diff
@@ -33,15 +33,20 @@ If you're using an older version of NorthstarProton, replace the information abo
-4. **If you have the game installed on Lutris:** right click _Titanfall 2_ > _Configure_ > _Game Options_ > Set _Executable path_ to _NorthstarLauncher.exe_
-5. **Otherwise:** click the `+` button in the top left > set the name to whatever and _Runner_ to _Wine_ > click on _Game options_ > set _Executable path_ to _NorthstarLauncher.exe_ then save.
+1. If you don't already have the game downloaded, you can use [this Lutris script](https://github.com/begin-theadventure/lutris-scripts/tree/main/lutris-scripts/Northstar/EA%20App).
+2. Install the latest version of Northstar using [FlightCore](../installing-northstar/northstar-installers#geckoeidechse-flightcore), [Viper](../installing-northstar/northstar-installers#0negal-viper), or do it manually
```
personally think we should be recommending manual first & foremost over auto installers

ultra-azu replied:
That part is copied from the Steam part of the guide. If you want I can change it there too though.

GeckoEidechse replied:
> personally think we should be recommending manual first & foremost over auto installers

Cringe take >:(

lizardwizardonthewall replied:
> > personally think we should be recommending manual first & foremost over auto installers
> 
> Cringe take >:(

![image](https://user-images.githubusercontent.com/116909003/236630466-c4f1c405-7585-4662-8bc3-2c528aeb7a2c.png)

lizardwizardonthewall commented on docs/steamdeck-and-linux/installing-on-steamdeck-and-linux.md:
```diff
@@ -33,15 +33,20 @@ If you're using an older version of NorthstarProton, replace the information abo
+   1. For manual install, download the latest version of Northstar from the [releases](https://github.com/R2Northstar/Northstar/releases) page
+   2. Then extract all contents of the file to your Titanfall 2 folder ( Right click _Titanfall 2_ > Click _Browse files_ )
+3. Add `-northstar` as a [launch option](../installing-northstar/troubleshooting.md#launch-opts) to Titanfall2
+4. Disable in-game overlay: In EA App, click on the hamburger menu on the top left > Click _Settings_ > Click _Application_ > Toggle _in-game overlay_ to OFF. In Origin, it's not necessary to turn off overlay, but doing so improves performance.
```
```suggestion
4. Disable in-game overlay: In EA App, click on the hamburger menu on the top left > Click _Settings_ > Click _Application_ > Toggle _in-game overlay_ to OFF.
```
we shouldn't be mentioning Origin anymore.

ultra-azu replied:
I added the mentions to Origin because Gecko mentioned it earlier.

GeckoEidechse replied:
Yup, I'd keep it as we still have a bunch of people using it...

lizardwizardonthewall replied:
Origin is deprecated, and we should treat it as such and stop mentioning it, keeping it alive will just cause more issues in the long run.

GeckoEidechse replied:
I don't fully disagree but the reality is that right now (especially on Linux) we still have a bunch of people using it. Removing Origin from the wiki should be done in separate PR IMO

lizardwizardonthewall commented on docs/steamdeck-and-linux/installing-on-steamdeck-and-linux.md:
```diff
@@ -33,15 +33,20 @@ If you're using an older version of NorthstarProton, replace the information abo
+{% endhint %}
 
-Now just launch the game through Lutris and you should be greeted with a Northstar welcome message upon entering the main menu.
+Now just launch the EA App/Origin through Lutris, open the game and you should be greeted with a Northstar welcome message upon entering the main menu.
```
```suggestion
Now just launch the EA App through Lutris, open the game and you should be greeted with a Northstar welcome message upon entering the main menu.
```
no Origin nonsense

itscynxx replied:
```suggestion
Now just launch EA App/Origin through Lutris, open the game and you should be greeted with a Northstar welcome message upon entering the main menu.
```
We should probably get to one decided way to refer to the EA App. Earlier in the page, its referred to as just `EA App` without saying `The EA App`. The troubleshooting page is also guilty of this issue but it should be changed to be one way, starting here preferably while there's already an open pr for a page with the issue

lizardwizardonthewall commented on docs/steamdeck-and-linux/installing-on-steamdeck-and-linux.md:
```diff
@@ -33,15 +33,20 @@ If you're using an older version of NorthstarProton, replace the information abo
+Now just launch the EA App/Origin through Lutris, open the game and you should be greeted with a Northstar welcome message upon entering the main menu.
 
-> **Note:** Origin might prompt you to log in and "set an installation folder for future downloads" on first launch. Just do those, close Origin, then launch the game again.
+> **Note:** If using Origin, it might prompt you to log in and "set an installation folder for future downloads" on first launch. Just do those, close Origin, then launch the game again.
```
```suggestion

```
_Origin begone_

ultra-azu commented:
So do I apply stygia's suggestions or do I leave it as is?

itscynxx commented:
> So do I apply stygia's suggestions or do I leave it as is?

I would leave as is (sadly)
People still love to cling to Origin for whatever reason and keeping it mentioned removes the small chance it confuses someone still using Origin

greengeckoz commented:
As far as I know, Origin is pretty much dead and can't be used. However I made a somewhat duplicate pull request on #194

lizardwizardonthewall commented:
> As far as I know, Origin is pretty much dead and can't be used. However I made a somewhat duplicate pull request on #194

There are workarounds to make Origin continue working (annoyingly enough).

GeckoEidechse commented:
Uh, what's the state on this?

GeckoEidechse commented:
bump
